packages/medusa/src/api/routes/admin/users/get-user.js

```diff
@@ -0,0 +1,13 @@
+export default async (req, res) => {
+  const { user_id } = req.params
+
+  try {
+    const userService = req.scope.resolve("userService")
+
+    let user = await userService.retrieve(user_id)
+    user = await userService.decorate(user, ["email", "name"])
```

Comment:
Admins are allowed to view anything so it isn't really necessary to decorate. 

At some point it may be relevant to have custom decoration for admin and store but for now let's just return the raw object.

Reply:
Yeah, you are right! Should I remove the decoration from all user admin routes?

Reply:
I think we might as well do that for now just to keep things consistent

Reply:
Noted!